x-pack/libbeat/licenser/license.go

```diff
@@ -27,7 +25,7 @@ import (
 // mode is the license in operation. (effective license)
 // status is the type installed is active or not.
 type License struct {
-	UUID        uuid.UUID   `json:"uid"`
+	UUID        string      `json:"uid"`
 	Type        LicenseType `json:"type"`
 	Mode        LicenseType `json:"mode"`
 	Status      State       `json:"status"`
```

Comment:
Down on line 55 there is a typo `s/avaiables/available/`.

Reply:
good catch fixed.